lib/strarray.c

```diff
@@ -359,12 +359,11 @@ EXPORTED char *strarray_join(const strarray_t *sa, const char *sep)
     return buf;
 }
 
-EXPORTED strarray_t *strarray_splitm(char *buf, const char *sep, int flags)
+EXPORTED void strarray_splitm(strarray_t *sa, char *buf, const char *sep, int flags)
```

Comment:
Its not critical, but could we have this be:

EXPORTED strarray_t *strarray_splitm(strarray_t *sa, char *buf, const char *sep, int flags)

And if we pass in a NULL array, we create a new one?

Reply:
yes, good idea